docs/time/index.rst

```diff
@@ -199,6 +199,7 @@ Format            Class                                         Example Argument
 byear        :class:`~astropy.time.TimeBesselianEpoch`          1950.0
 byear_str    :class:`~astropy.time.TimeBesselianEpochString`    'B1950.0'
 cxcsec       :class:`~astropy.time.TimeCxcSec`                  63072064.184
+galexsec     :class:`~astropy.time.TimeGalexSec`                758738047.995
```

Comment:
Maintain alphabetical order and put between `fits` and `gps`.

Though I notice somehow `datetime64` got left at the bottom of the list, so you might as well fix that now as well.

Reply:
My intend was a logical order (group the satellites together) since it wasn't fully alphabetic before, but I don't feel strongly either way. Will reorder.